planning/surround_obstacle_checker/src/debug_marker.cpp

```diff
@@ -36,13 +36,22 @@ using tier4_autoware_utils::createMarkerScale;
 using tier4_autoware_utils::createPoint;
 
 SurroundObstacleCheckerDebugNode::SurroundObstacleCheckerDebugNode(
-  const double base_link2front, const rclcpp::Clock::SharedPtr clock, rclcpp::Node & node)
-: base_link2front_(base_link2front), clock_(clock)
+  const Polygon2d & ego_polygon, const double base_link2front,
+  const double & surround_check_distance, const geometry_msgs::msg::Pose & self_pose,
+  const rclcpp::Clock::SharedPtr clock, rclcpp::Node & node)
+: ego_polygon_(ego_polygon),
+  base_link2front_(base_link2front),
+  surround_check_distance_(surround_check_distance),
```

Comment:
This module has another offset parameter `surround_check_recover_distance` to clear the stop judgement.
So, I think it is better and useful to output another polygon with this parameter. What do you think ?

The detail algorithm is shown in README.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/surround_obstacle_checker/README.md

https://github.com/autowarefoundation/autoware.universe/blob/9ff6235c154a675cedfafe73b6d8eed840903d78/planning/surround_obstacle_checker/src/node.cpp#L152-L153

Reply:
I say since it is for debugging purposes the more is better :) 

I will add it asap. Thank you for the suggestion!